client/src/Features/StudentView/StudentPage.tsx

```diff
@@ -41,18 +40,4 @@ const StudentPageStyles = ({ theme }: IThemeOnlyProps): StudentPageStyles => ({
   ]
 });
 
-const pageWrapperStyles = ({ theme }: IThemeOnlyProps): Partial<PageWrapperStyles> => ({

```

Comment:
removed these styles so as to have uniformity between teacher view and student view